.travis.yml

```diff
@@ -1,6 +1,26 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-sudo: true
+sudo: required
+dist: trusty
+before_install:
+    - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu/ trusty main universe"
+    - sudo apt-get update -qq
+    - sudo apt-get install libgcrypt20 liblzma5 libselinux1
+    - wget "http://archive.ubuntu.com/ubuntu/ubuntu/pool/main/s/systemd/libsystemd0_219-7ubuntu3_amd64.deb"
+    - wget "http://archive.ubuntu.com/ubuntu/ubuntu/pool/main/s/systemd/libsystemd-dev_219-7ubuntu3_amd64.deb"
+    - sudo dpkg -i libsystemd0_219-7ubuntu3_amd64.deb libsystemd-dev_219-7ubuntu3_amd64.deb
+    - git clone https://github.com/xapi-project/xs-opam && cd xs-opam && sh into_repo.sh && cd -
```

Comment:
@mseri I had to do exactly the same thing, before I removed coverage from `.travis.yml`. It happened because the [`ocaml-travis-coveralls`](https://github.com/simonjbeaumont/ocaml-travis-coveralls/) script is trying to delete `bisect*` files and dirs at one point, and fails for the non-empty `bisect_ppx` directory from `xs_opam`.